modules/servers/openprovider/lib/openprovider_api.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'response.php';
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'last_request.php';
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'command_mapping.php';
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'api_configuration.php';
+require_once __DIR__ . DIRECTORY_SEPARATOR . 'params_builder/params_creator_factory.php';
+
+use Openprovider\Api\Rest\Client\Base\Configuration;
+use GuzzleHttp6\Client as HttpClient;
+use Symfony\Component\Serializer\NameConverter\CamelCaseToSnakeCaseNameConverter;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+use Symfony\Component\Serializer\Serializer;
+
+class OpenProviderApi
+{
+    const API_CLIENT_NAME = 'whmcs-plesk-1';
+    const API_URL = 'https://api.openprovider.eu';
```

Comment:
not used constant
it's better to switch environments via configuration file

Reply:
I'm not sure if this is really necessary, because this URL is not needed by users, and it is a permanent URL